userspace/libsinsp/filterchecks.cpp

```diff
@@ -3164,7 +3167,10 @@ uint8_t* sinsp_filter_check_event::extract(sinsp_evt *evt, OUT uint32_t* len, bo
 				evname = (uint8_t*)evt->get_name();
 			}
 
-			*len = strlen((char *) evname);
+			if(evname)
```

Comment:
No, I thought it might be possible that evt->get_name() could return NULL. After further inspection that seems fairly unlikely. I can take these checks out.